pwiz_tools/Skyline/Test/CodeInspectionTest.cs

```diff
@@ -215,6 +219,50 @@ List<string> CheckFormsWithoutTestRunnerLookups()
             return missing;
         }
 
+        /// <summary>
+        /// We have some code, especially in commandline, that expects certain error messages to start with "Error:" or the L10N equivalent
+        /// </summary>
+        void InspectConsistentErrorMessages(List<string> errors)
+        {
+            var currentCulture = Thread.CurrentThread.CurrentUICulture;
+            try
+            {
+                Thread.CurrentThread.CurrentCulture = Thread.CurrentThread.CurrentUICulture = new CultureInfo("en"); // We want to compare against the "en" resources
+                var resourceSet = Skyline.Properties.Resources.ResourceManager.GetResourceSet(Thread.CurrentThread.CurrentUICulture, true, true);
+
+                // Build the string of resources.
+                foreach (var resource in resourceSet)
```

Comment:
Couldn't you just use First() to get the first value instead of using a foreach loop and increasing the indenting. That also has the benefit of failing if for some reason the code to get the "en" language fails. Otherwise, it looks like the code would just fail to do any checking at all and silently allow anything.

Reply:
Let me clean up the comments here and see if it makes more sense to you then.